work_test.go

```diff
@@ -236,23 +237,21 @@ func TestLockJobAdvisoryRace(t *testing.T) {
 		return conn
 	}
 	getBackendID := func(conn *pgx.Conn) int32 {
```

Comment:
can you also rename `getBackendID` to `getBackendPID` for consistency with the variable rename (which I agree was the right choice)?